### PR TITLE
Fixes #11648: Unexpected report for Send inventories to CMDB after #11257

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/11648

### CODE REVIEW

File: techniques/system/distributePolicy/1.0/propagatePromises.st [hunk 1]
```diff
@@ -250,7 +250,7 @@ bundle agent sendInventoryToCmdb
     pass3.rudder_inventory_processing_error::
       "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "&TRACKINGKEY&", "Send inventories to CMDB", "None", "Some inventories failed to add successfully to Rudder");
 
-    pass3.((root_server|role_rudder_relay_top).!rudder_inventory_processing_repaired.!rudder_inventory_processing_error)::
+    pass3.((root_server|role_rudder_relay_top).!rudder_inventory_processing_repaired.!rudder_inventory_processing_error.!disable_upload)::
       "any" usebundle => rudder_common_report("DistributePolicy", "result_success", "&TRACKINGKEY&", "Send inventories to CMDB", "None", "No inventory to send");
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/propagatePromises.st]: Could you rename CMDB to Rudder server, or something more understandable?

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**